src/System.Security.Cryptography.X509Certificates/src/System.Security.Cryptography.X509Certificates.csproj

```diff
@@ -86,6 +107,14 @@
     <Compile Include="System\Security\Cryptography\X509Certificates\X509SubjectKeyIdentifierExtension.cs" />
     <Compile Include="System\Security\Cryptography\X509Certificates\X509SubjectKeyIdentifierHashAlgorithm.cs" />
     <Compile Include="System\Security\Cryptography\X509Certificates\X509VerificationFlags.cs" />
+    <Compile Include="System\Security\Cryptography\X509Certificates\Asn1\BasicConstraintsAsn.cs" />
+    <Compile Include="System\Security\Cryptography\X509Certificates\Asn1\KeyUsageFlagsAsn.cs" />
+    <Compile Include="System\Security\Cryptography\X509Certificates\Asn1\TimeAsn.cs" />
+    <Compile Include="System\Security\Cryptography\X509Certificates\Asn1\ValidityAsn.cs" />
+    <Compile Include="System\Security\Cryptography\X509Certificates\Asn1\TbsCertificateAsn.cs" />
+    <Compile Include="System\Security\Cryptography\X509Certificates\Asn1\ExtensionsAsn.cs" />
+    <Compile Include="System\Security\Cryptography\X509Certificates\Asn1\X509ExtensionAsn.cs" />
+    <Compile Include="System\Security\Cryptography\X509Certificates\Asn1\CertificateAsn.cs" />
```

Comment:
Not sorted, I know...

Reply:
... and CertificateAsn, TbsCertificateAsn, ValidityAsn, TimeAsn, ExtensionsAsn and X509ExtensionAsn should go to common code. They are already present in Pkcs (X509ExtensionAsn) or in the common code in simplified version (Certificate, TbsCertificateLite).